src/Oliver.Client/appsettings.json

```diff
@@ -10,16 +10,16 @@
     "BaseUrl": "https://localhost:5001/"
   },
   "Client": {
-    "DefaultFolder": "d:\\temp\\Oliver",
+    "DefaultFolder": "../../../../workFolder", // sln folder
```

Comment:
Hmm, `JSON` format doesn't support comments, so how does it work? 🤔

Reply:
yes, it would work, because dotnet expects JSONWithComments format))